lib/druid/aggregation.rb

```diff
@@ -3,13 +3,13 @@ class Aggregation
     include ActiveModel::Model
 
     attr_accessor :type
-    validates :type, inclusion: { in: %w(count longSum doubleSum min max javascript cardinality hyperUnique) }
+    validates :type, inclusion: { in: %w(count longSum doubleSum floatSum longMin longMax doubleMin doubleMax floatMin floatMax hyperUnique) }
 
     attr_accessor :name
     validates :name, presence: true
 
     class FieldnameValidator < ActiveModel::EachValidator
-      TYPES = %w(count longSum doubleSum min max hyperUnique)
+      TYPES = %w(longSum doubleSum floatSum longMin longMax doubleMin doubleMax floatMin floatMax)
```

Comment:
Why have you removed `count` and `hyperUnique` types?

Reply:
@pikislabis My Bad.. added missing aggregations.